packages/app/src/cli/models/extensions/specifications/ui_extension.ts

```diff
@@ -107,6 +107,12 @@ Please check the module path for ${target}`.value,
     }
   }
 
+  if (
+    uniqueTargets.indexOf('customer-account.order.page.render') >= 0 &&
+    uniqueTargets.indexOf('customer-account.page.render') >= 0
+  ) {
+    errors.push(`You can't have both 'customer-account.order.page.render' and 'customer-account.page.render'`)
```

Comment:
Is it possible to show a link to a public doc which shows why this is not doable?

Reply:
we don't have any doc yet....